.vscode/launch.json

```
@@ -0,0 +1,28 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "(ctest) Launch",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${command:cmake.launchTargetPath}",
+            "args": [],
+            "stopAtEntry": false,
+            "cwd": "${fileDirname}",
+            "environment": [
+                {
+                    // add the directory where our target was built to the PATHs
+                    // it gets resolved by CMake Tools:
+                    "name": "PATH",
+                    "value": "${env:PATH}:${command:cmake.getLaunchTargetDirectory}"
+                },
+            ],
+            "externalConsole": false,
+            "internalConsoleOptions": "openOnSessionStart",
+            "MIMode": "lldb"
```

I guess some people might prefer gdb over lldb, especially on Ubuntu since it comes as default in `build-essential` metapackage.
Apparently it can be setup on a per-OS basis, [see here](https://code.visualstudio.com/docs/cpp/launch-json-reference#_launchcompletecommand)
```suggestion
            "linux": {
                "MIMode": "gdb",
              },
              "osx": {
                "MIMode": "lldb"
              },
```

---

Since we changed the cwd for the tests, we had to set the settings.json for TestMate.

---

See `settings.json`